### PR TITLE
Mark/update notification settings design

### LINKED ISSUE #17087: Update Notification Settings design
Update the design in the OSS notification settings page with `Scenario #5` in the prototype:

https://www.figma.com/proto/SBtckxQGXX9V1jXLUgwRYp/01_02_NOTIFICATIONS?page-id=0%3A1&node-id=120%3A1522&viewport=865%2C929%2C0.15&scaling=min-zoom&starting-point-node-id=120%3A1812&show-proto-sidebar=1

* Move the save button under the card
* Leave the test button visible (and disable when webhooks is empty)
* Improve documentation panel
* Ensure switches are disabled when webhook is blank

### PR DESCRIPTION
**Pull request by m19berger**
## What
Closes [#17087](https://github.com/airbytehq/airbyte/issues/17087)

## How
Notification settings design page redesign 

- Moved the `Save changes` button under the card
- Changed the behavior of the notification form
- Added button `Need help with Webhook URL?`
- Created notification guide panel

[Figma design](https://www.figma.com/proto/SBtckxQGXX9V1jXLUgwRYp/01_02_NOTIFICATIONS?node-id=120%3A1654&starting-point-node-id=120%3A1812&scaling=scale-down)

## Loom
Updated Loom with the latest changes:
https://www.loom.com/share/b8ecd9f3a3b04a7f8c5a93fc696f80d2

### CODE REVIEW

File: airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx
Comment (edmundito) [inline airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx:52]: There should be a feedback message when the notification test succeeds or fails, the same for trying to save the connection. There is no feedback message, even on the main branch. Would it be possible to fix this here?

File: airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx
Comment (edmundito) [inline airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx:89]: This link should be referenced from `src/utils/links.ts`

File: airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx
Comment (edmundito) [inline airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx:102]: This link should be referenced from `src/utils/links.ts`

**Review by edmundito — CHANGES_REQUESTED** [review]
The code is looking really good. I did some local testing and there are a couple of areas that can be improved in terms of indicating success and error. The rest is mostly style issues.

Additionally, we are trying to move away from using the public folder (#15956) and instead the `webhook-guide.png` image should be alongside the webhook component.

File: airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx
Comment (dizel852) [reply airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx:53]: Agree with @edmundito. Also tested locally, and we indeed need to show the user the webhook URL testing (was it successful or not)

@matter-q yeah, we have such a service - https://github.com/airbytehq/airbyte/blob/9db329eb6c9e2cd3e7815b46fd2c651dfa3c3b14/airbyte-webapp/src/hooks/services/Notification/NotificationService.tsx#L12

Please note, there can be 2 possible cases when a testing hook request can be failed:
- 4xx status response from BE (if we have sent the totally wrong URL)
- 2xx status response from 3rd party hook service (need to check the response)

I've used the https://webhook.site/ service for testing webhooks

**Review by dizel852 — CHANGES_REQUESTED** [review]
Overall - code LGTM.
Was reviewing after @edmundito comment fixes.

Left a comment regarding NoficationsService.

File: airbyte-webapp/src/locales/en.json
Comment (matter-q) [reply airbyte-webapp/src/locales/en.json:492]: @andyjih Could you also provide messages for the `Test` action button when it's a `success` and `fail` operation?

Comment (matter-q) [reply airbyte-webapp/src/locales/en.json:497]: My question is covered by Nicolas Philippot

Comment (tealjulia) [reply airbyte-webapp/src/locales/en.json:497]: Decided on Notification Settings during standup

File: airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx
Comment (edmundito) [inline airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx:81]: This custom logic should not be needed here. The message should be visible until the user presses "test" again.

**Review by edmundito — COMMENTED** [review]
Had some notes about the error handling

File: airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx
Comment (matter-q) [reply airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx:250]: We need to clarify the behavior of such a form. 

Currently, if the user entered a **valid** URL, he can test it (button `Test`) and save changes (button `Save changes`). Because the form is valid. Users can't test or save changes if an entered string does not match the URL format. Also, the user can test his webhook without saving.

File: airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx
Comment (matter-q) [reply airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx]: Fixed. Thanks :smile:

File: airbyte-webapp/src/utils/links.ts
Comment (matter-q) [reply airbyte-webapp/src/utils/links.ts]: Fixed :smile:

File: airbyte-webapp/src/components/icons/PlayIcon.tsx
Comment (matter-q) [reply airbyte-webapp/src/components/icons/PlayIcon.tsx]: Fixed :smile:

File: airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx
Comment (matter-q) [reply airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx]: Changed to the left side. Thanks :grinning:

**Review by dizel852 — APPROVED** [review]
Code LGTM, tested locally.
Let's wait for final approval from @edmundito 

@edmundito question:
Currently, we show the input error in case the test is failed. 
Why we decided do not to show the message in case of successful testing?

**Comment by matter-q** [comment]
Waiting for final UI decision from Niko :grinning:

**Comment by dizel852** [comment]
@Upmitt

**Review by edmundito — APPROVED** [review]
Tested locally and testing/save/validation rules work as expected!
Tested links on doc and both navigate to the correct page

**Pull request merged**
